src/lib/snark_worker/intf.ml

```diff
@@ -20,165 +15,18 @@ module type Inputs_intf = sig
     val worker_wait_time : float
   end
 
-  val perform_single :
-       Worker_state.t
-    -> message:Mina_base.Sok_message.t
-    -> (Transaction_witness.Stable.Latest.t, Ledger_proof.t) Work.Single.Spec.t
-    -> (Ledger_proof.t * Time.Span.t) Deferred.Or_error.t
-end
-
-module type Rpc_master = sig
```

Comment:
It would be nice to keep the interface as before.

Reply:
Basically I have the same question as https://github.com/MinaProtocol/mina/pull/17040#discussion_r2069795683

PPX make it hard to have a compilable interface. 
![image](https://github.com/user-attachments/assets/34e66a74-8c39-4e38-bb2a-7c3a8fe5d964)

I could in theory put all definitions generated by the ppx to the MLI but that seems bad because it's just bunch of biolerplates

Reply:
I figured this out. But all type annotations should go to later PR. As o.w. dune refuse to compile any module exposing unused definitions.